resources/manifests/kubelet-pod-checkpointer-psp-cluster-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
(repasting comments from matheus patch): Why not use RoleBinding and the kube-system namespace?